chatbot.py

```diff
@@ -38,28 +39,26 @@ def get_close_matches(topic, titles):
     return matches
 
 
-# display strings in a numbered list
 def numbered_print(strings):
-    count = 1
+    '''Display strings in a numbered list.'''
     final = ""
-    for s in strings:
-        final += str(count) + '. ' + ", ".join(s) + "\n"
-        count += 1
+    for i, s in enumerate(strings):
+        final += str(i+1) + '. ' + ", ".join(s) + "\n"
     return final
 
 
 # state 1
 def get_topic(model_endpoint, topic, titles):
+    if topic.lower() == "stop":
+        return "Thank You for using QnAit, Hope your questions were answered!", 5, {}
+
     # hardcoded fun :)
     if topic == "What is the meaning of life?":
-        return "42\nIs there another topic you are curious about?", 1, {}
-    
-    if topic == "No":
-        return "Thank You for using QnAit, Hope your questions were answered!", 5, {}
+        return "42\n\nIf you are curious about another topic, reply with the topic.", 1, {
```

Comment:
I think this `{` doesn't have a `}` to close it

Reply:
Thanks I verified it runs now